### PR TITLE
Fix and improvements for mapwize-maps integration

### PR DESCRIPTION
**Pull request by Sparksx**

### CODE REVIEW

File: src/header/searchResults/searchResults.html [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
+      <div class="progress-bar progress-bar-striped progress-bar-animated" role="progressbar" style="width: 100%;"
+        aria-valuenow="25" aria-valuemin="0" aria-valuemax="100"></div>
     </div>
+  </div>
 
-    <div id="mwz-search-results-container" class="list-group list-group-flush"></div>
-</div>
\ No newline at end of file
+  <div id="mwz-use-user-location">
+    <button class="btn btn-link btn-block">Use my location</button>
```
Comment (mathieugerard) [inline src/header/searchResults/searchResults.html]: This needs to be translated

File: src/locales/en.locale.json [hunk 2]
```diff
@@ -2,16 +2,11 @@
...
-  "choose_destination":"Choose destination...",
-
+  "choose_starting_or_click_point": "Choose starting point, or click on the map...",
+  "choose_destination_or_click_point": "Choose destination, or click on the map...",
+  "choose_destination": "Choose destination...",
   "on_floor": "On floor <%= floor %>",
-
   "coordinates": "Coordinates",
-
+  "location": "Current location",
```
Comment (mathieugerard) [inline src/locales/en.locale.json]: "current_location"

**Comment by mathieugerard** [comment]
If a user location with floor is available, it should be used by default for the "to"

**Comment by mcolmant** [comment]
There is a conflict to resolve, then we can merge.

**Pull request merged**